Applications/Scale/test/testScale.cpp

```diff
@@ -184,3 +185,46 @@ void scaleGait2354_GUI(bool useMarkerPlacement)
 
     delete subject;
 }
+
+void scaleModelWithLigament()
+{
+    // SET OUTPUT FORMATTING
+    IO::SetDigitsPad(4);
+
+    std::string setupFilePath("");
+    ScaleTool* scaleTool;
+    Model* model;
+
+    // Remove old model if any
+    FILE* file2Remove = IO::OpenFile(setupFilePath + "toyLigamentModelScaled.osim", "w");
+    fclose(file2Remove);
```

Comment:
You're correct, it empties the file rather than removes it. I'll update the comment(s) accordingly.